### ISSUE TITLE
[1.10.2, 1.9.4] HQM no longer generates config files

### ISSUE THREAD

**Issue by TriageONE** [lorddusk/HQM#325]
I have been working on a modpack for quite some time now and have realized that my config files were no longer being read, or written at all. I continually and aggressively screw with configs and have tried all 4 versions avail on curseforge and not one of them will generate a single config file. 

All it does is give me an empty quest book and wont load any quests. It doesn't provide the option for changing the variable for editmode in [now nonexistent] hqmedit.cfg 

The only folder being generated is the remote folder and there is never anything inside, its always empty. Adding an empty editmode.cfg file, crashes the game [not in remote, under .config/hqm/] with the standard NullPointer error. 

I have:

- changed perms so anyone can read/write.
- watched the console for multimc; no errors or crashes thrown.
- watched for conflictions; there are none.
- disabled all security temporarily, no luck. 

Any help?

**Comment by canitzp** [comment]
I've never seen any comparable issue at this issue tracker. Can you provide some more information about your Environment like versions and a log file?

**Comment by TriageONE** [comment]
I have done some testing in between my rick and morty binge and set up a new environment with only HQM and found that It may be the mudpack environment that I'm running, ill post logs


[fml-client-latest.txt](https://github.com/lorddusk/HQM/files/999079/fml-client-latest.txt)


That's the log file. Too bad most of its enderIO failing at finding its translations. . 

Running 1.10.2. Forge 12.18.3.2185 LWJGL 2.9.1, not that that matters. 

The only thing that i did right before it started acting up was install veinminer. Ill continue testing, maybe there's an incompatibility issue that I cant explain.

**Comment by TriageONE** [comment]
Just opened another instance with MultiMC and the configs generated

Purged current configs and the configs generated, [Copied over to another file]

ANOMALY: Found [BetterFPS1.4.4-1.10] and [Optifine U D8] latest, banishing BetterFPS, killing the configs for said banished mod.

Restore old configs, keep quests in temporary folder, delete hqm config file in confic dir. 

restarting minecraft..
.....

Still no success, just a readme file.

------EDIT------

So I tried disabling half the mods, then moving the way down the list and re-enabling them as I startup/shut down Minecraft. I disabled half, and the configs showed up! Miracle!
It gets weird quick, because I got to the bottom with enabling the rest of the mods after deleting the hqm config folder and waiting and apparently it fucking fixed itself.

I have no clue what caused this but ill give you any info you want. My guess is that there was some sort of weird incompatability that resolved itself. In a nutshell this is something I cant explain and I fixed it without knowing how. 

Just tested again.. I have no idea what I did wrong, this is so odd
